packages/oscal-react-library/src/components/OSCALCatalog.tsx

```diff
@@ -25,7 +39,10 @@ export default function OSCALCatalog(props) {
         urlFragment={urlFragment}
       />
 
-      <OSCALCatalogGroups groups={catalog.groups} urlFragment={urlFragment} />
+      <OSCALCatalogGroups
+        groups={catalog.controls ? [defaultGroup, ...(catalog.groups ?? [])] : catalog.groups}
+        urlFragment={urlFragment}
+      />
```

Comment:
This will be awkward when there are no groups at all. Realizing that I don't know the names of all the components off the top of my head, what if we did:

```tsx
catalog.groups
  ? <OSCALCatalogGroups groups={catalog.controls ? [defaultGroup, ...(catalog.groups ?? [])] : catalog.groups} urlFragment={urlFragment} />
  : catalog.controls
  ? <OSCALCatalogControlsList controls={catalog.controls} />
  : undefined
```

Reply:
My initial idea was similar to this. The list of controls displayed in the catalog is pretty tightly coupled to groups. Another option would be to display the controls like how we do in profile, however this also would not be super clean and would look a bit weird. 

I think that having a single group, while a bit strange, is the cleanest way to do this at the moment.